.github/workflows/dispatch_workflows.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       - name: Dispatch workflows
         shell: bash
-        run: docker run -e GITHUB_TOKEN artifactory.algol60.net/csm-docker/stable/hms-build-workflow-dispatcher:1.0.0
+        run: docker run -e GITHUB_TOKEN artifactory.algol60.net/csm-docker/unstable/hms-build-workflow-dispatcher:1.1.0-20220701221252.17f5392
```

Comment:
This will need to be updated before this PR merges, with the expected docker image tag.